frontend/tests/features/genes/geneSets.test.ts

```diff
@@ -1,6 +1,5 @@
 import { expect, Page, test } from "@playwright/test";
 import { goToPage } from "tests/utils/helpers";
-import { getTestID } from "tests/utils/selectors";
```

Comment:
please help delete this file completely now that https://github.com/chanzuckerberg/single-cell-data-portal/issues/2121 is merged 🙏 Thank you!